climada/hazard/trop_cyclone.py

```diff
@@ -892,26 +915,26 @@ def _B_holland_1980(gradient_winds, penv, pcen):  # pylint: disable=invalid-name
 
     Parameters
     ----------
-    gradient_winds : np.array
+    gradient_winds : np.ndarray
         Maximum gradient-level wind speeds (m/s) of the tropical cyclone. If your data are maximum
         surface wind speeds (e.g. from IBTrACS), make sure to subtract translational wind speed and
         convert to gradient-level winds first.
-    penv : np.array
+    penv : np.ndarray
         Environmental pressure (hPa).
-    pcen : np.array
+    pcen : np.ndarray
         Central pressure (hPa).
 
     Returns
     -------
-    B : np.array
+    B : np.ndarray
         Holland b-value
     """
     # the factor 100 is from conversion between mbar and pascal
     pdelta = 100 * (penv - pcen)
     hol_b = gradient_winds**2 * np.exp(1) * RHO_AIR / np.fmax(np.spacing(1), pdelta)
     return np.clip(hol_b, 1, 2.5)
 
-def _x_holland_2010(d_centr, r_max, v_max_s, hol_b, close_centr, v_n=17.0, r_n=300e3):
+def _x_holland_2010(d_centr, r_max, v_max_s, hol_b, close_centr, v_n=17.0, r_n=300):
```

Comment:
This was a bug in the H2010 model: We wrongly assumed that all variables are in SI units, but actually, we currently use km for distances everywhere.